en/cpp/contributing/plugins.md

```diff
@@ -119,7 +122,7 @@ message SetReturnToLaunchAltitudeResponse {
 }
 ```
 
-> **Note** that requests can defined SYNC, ASYNC, or BOTH using `option (mavsdk.options.async_type) = ...;`.

```

Comment:
@hamishwillee Can this be replaced with "::: note" block like PX4 User Doc? If so, I think it would be better to use that system :thinking: